tiny_gltf_loader.h

```diff
@@ -897,6 +940,41 @@ static bool ParseNumberProperty(double *ret, std::string *err,
   return true;
 }
 
+static bool ParseJSONProperty(std::map<std::string, std::string> *ret, std::string *err,
+                              const picojson::object &o,
+                              const std::string &property,
+                              bool required)
+{
+  picojson::object::const_iterator it = o.find(property);
+  if(it == o.end())
+  {
+    if (required) {
+      if(err) {
+        (*err) += "'" + property + "' property is missing. \n'";
+      }
+    }
+    return false;
+  }
+
+  if(!it->second.is<picojson::object>()) {
+    if (required) {
+      if (err) {
+        (*err) += "'" + property + "' property is not an array.\n";
```

Comment:
error message looks wrong; you're testing `is<picojson::object>` not `is<picojson::array>`